CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# dbt_netsuite v1.1.0
+
+## Schema/Data Change
+**7 total changes • 2 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ------------- | ----------- | ----| --- | ----- |
+| `netsuite2__entity_subsidiary_relationships` | New model | | | Unified view combining customer and vendor subsidiary relationships |
+| `netsuite2__transaction_details` | New columns | | `nexus_id`<br>`nexus_country`<br>`nexus_state`<br>`tax_agency_id`<br>`tax_agency_alt_name`<br>`is_nexus_override`<br>`is_tax_details_override`<br>`tax_point_date`<br>`is_tax_point_date_override` | Nexus-related tax information when var `netsuite2__using_nexuses` is enabled |
+| `stg_netsuite2__nexuses` | New model | | | Provides access to Netsuite tax nexus data |
+| `stg_netsuite2__customer_subsidiary_relationships` | New model | | | Maps customers to their associated subsidiaries |
+| `stg_netsuite2__vendor_subsidiary_relationships` | New model | | | Maps vendors to their associated subsidiaries |
+| `stg_netsuite2__transactions` | New columns | | `nexus_id`<br>`is_nexus_override`<br>`is_tax_details_override`<br>`tax_point_date`<br>`is_tax_point_date_override` | Nexus-related tax information when var `netsuite2__using_nexuses` is enabled.<br><br>**Breaking change**: If you already include any of these fields via the [passthrough columns](https://github.com/fivetran/dbt_netsuite/blob/main/README.md#passing-through-additional-fields) variable `transactions_pass_through_columns`, remove them from the list to avoid duplicate column errors. |
```

Comment:
Ah still gotta document these in the stg.yml 
<img width="982" height="562" alt="image" src="https://github.com/user-attachments/assets/8024a8c7-c097-4db1-b668-8174ea8f61fe" />

Reply:
added and regenerated docs